thrall/app/lib/ElasticSearch.scala

```diff
@@ -117,7 +117,23 @@ object ElasticSearch extends ElasticSearchClient with ImageFields {
     }
   }
 
-def updateImageExports(id: String, exports: JsValue)(implicit ex: ExecutionContext): List[Future[UpdateResponse]] =
+  def updateImageLeases(id: String, leaseByMedia: JsValue, lastModified: JsValue)(implicit ex: ExecutionContext) : List[Future[UpdateResponse]] = {
+    prepareImageUpdate(id){ request =>
+      request.setScriptParams( Map(
+        "leaseByMedia" -> asGroovy(leaseByMedia),
+        "lastModified" -> asGroovy(lastModified)
+      ).asJava)
+        .setScript(
+          replaceLeasesScript +
+            updateLastModifiedScript,
+          scriptType)
+      .executeAndLog(s"updating leases on image $id with: $leaseByMedia")
```

Comment:
this is noisy (prints out the whole media lease) 

It was helpful for me but maybe not in the long term?